### ISSUE TITLE
"lime build android" fails on icon processing  

### ISSUE THREAD

**Issue by nitrobin** [openfl/lime#433]
lime 2.4.0
Build fails if project.xml contains "icon" elements.  
i.e.
```
<icon path="res/icon/android_512.png" size="512"/>
```
```
Called from ? line 1
Called from CommandLineTools.hx line 1359
Called from CommandLineTools.hx line 25
Called from CommandLineTools.hx line 126
Called from CommandLineTools.hx line 579
Called from lime/project/PlatformTarget.hx line 70
Called from lime/tools/platforms/AndroidPlatform.hx line 328
Called from lime/tools/helpers/IconHelper.hx line 83
Called from /usr/lib/haxe/std/neko/_std/sys/io/File.hx line 45
Called from /usr/lib/haxe/std/haxe/io/Output.hx line 105
Uncaught exception - Invalid field access : length
```

**Comment by Anheurystics** [comment]
I also get this problem, using the latest Haxe 3.2.0 release. It does not seem to show up when using 3.2.0-rc.2, though.

**Comment by lewislepton** [comment]
this problem is also happening to me on mac. but even when using the rc2 version does not work or fix the problem

**Comment by eduardonunesptoptal** [comment]
Failed here something similar: 

Lime 2.4.1
Haxe 3.1.3

```
lime test neko -debug
Warning: Could not find template file: neko/bin/neko-Mac64
Called from ? line 1
Called from CommandLineTools.hx line 1359
Called from CommandLineTools.hx line 25
Called from a C function
Called from CommandLineTools.hx line 126
Called from CommandLineTools.hx line 579
Called from lime/project/PlatformTarget.hx line 77
Called from lime/tools/platforms/MacPlatform.hx line 110
Called from lime/tools/helpers/NekoHelper.hx line 21
Called from /usr/lib/haxe/std/neko/_std/sys/io/File.hx line 34
Called from /usr/lib/haxe/std/neko/_std/sys/io/File.hx line 30
Uncaught exception - Invalid field access : __s```

**Comment by jgranick** [comment]
@eduardonunesptoptal It sounds like you are having a different problem. For some reason, it is not finding that template file. Could you see if you have it under "/usr/lib/haxe/lib/lime/2,4,1/lime/templates/neko/bin"?

Perhaps you have a bad install, or there's a chance that you have things installed on a case-sensitive drive?

**Comment by jgranick** [comment]
(closing this issue as the icon processing problem was resolved in 2.4.1)

**Comment by eduardonunesptoptal** [comment]
Sure, it's was a bad install, after resinstall everything is ok. Thanks
